profiles/base.yaml

```diff
@@ -160,6 +160,10 @@ cambrinth_cap: 32
 stored_cambrinth: false
 prep_scaling_factor: 0.98
 kneel_khri: false
+cambrinth_items:
+- name:
```

Comment:
Is this an empty cambrinth declaration? Will having this in `base.yaml` mean the `cambrinth_items` logic will be used immediately for everyone (since the array now has an item in it)?

Reply:
I was worried about that, but I tested it out, and the `if @settings.cambrinth_items` seemed to be returning false even with the array in the base.yaml, presumably since the hash was undefined.

Reply:
I ran this:
```
if get_settings.cambrinth_items
  echo 'true'
else
  echo 'false'
end
```
and got 'true'. That implies to me (though I haven't tested it) that the old style cambrinth data wouldn't be copied over and used.

Reply:
I just checked, and you're right -- I'm not sure why it was behaving differently for me before.  I updated to check for `@settings.cambrinth_items[0]['name']` specifically, which still returns false with the empty cambrinth declaration I added to base.yaml.